src/wp-includes/fonts/class-wp-font-face.php

```diff
@@ -229,7 +229,7 @@ private function validate_font_face_declarations( array $font_face ) {
 	private function get_style_element() {
 		$attributes = $this->generate_style_element_attributes();
 
-		return "<style id='wp-fonts-local'{$attributes}>\n%s\n</style>\n";
+		return "<style class='wp-fonts-local'{$attributes}>\n%s\n</style>\n";
```

Comment:
Here and throughout...

```suggestion
		$id_attribute = wp_unique_prefixed_id( 'wp-fonts-local-' );
		return "<style id='$id_attribute' class='wp-fonts-local'{$attributes}>\n%s\n</style>\n";
```